src/main/java/com/greenfox/dramacsoport/petclinicbackend/controllers/appUser/auth/RegisterController.java

```diff
@@ -3,42 +3,36 @@
 import com.greenfox.dramacsoport.petclinicbackend.dtos.register.RegisterRequestDTO;
 import com.greenfox.dramacsoport.petclinicbackend.services.appUser.auth.AuthService;
 import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import lombok.SneakyThrows;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
-import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
 import org.springframework.validation.BindingResult;
-import org.springframework.web.bind.annotation.PostMapping;
-import org.springframework.web.bind.annotation.RequestBody;
-import org.springframework.web.bind.annotation.ResponseBody;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.*;
 
 @RestController
+@RequiredArgsConstructor
+@RequestMapping("api/v1/auth")
 public class RegisterController {
 
-    @Autowired
-    private AuthService authService;
+    private final AuthService authService;
 
     private final Logger logger = LoggerFactory.getLogger(RegisterController.class);
 
+    @SneakyThrows
```

Comment:
You do not need to add the SneakyThrows annotation, you can just add the throws part to the signature, the ControllerAdvice will handle the rest.